av1an-core/Cargo.toml

```diff
@@ -44,6 +44,10 @@ path_abs = "0.5.1"
 av-scenechange = "0.7.0"
 y4m = "0.7.0"
 
+[dependencies.ffmpeg-next]
```

Comment:
Can we use ffmpeg-next but refer to it as ffmpeg by changing the package name in cargo? Specify the dependency as ffmpeg but add a property `package = "ffmpeg-next"`

Reply:
I attempted that, and it was easier to just use `ffmpeg_next` because we have a module called `ffmpeg` in av1an and that becomes confusing to both devs and the compiler.

Reply:
ah I see, makes sense. what do you think of calling it `avcodec` then, or do you prefer `ffmpeg_next`?

Reply:
I'm not a fan of using `avcodec` since technically that's just one part of ffmpeg. I also don't love how `ffmpeg_next` is kind of verbose, but I'm not sure of a great way to shorten it. 🤔

Reply:
I know there's another crate called `ffav` (but then, it might be confused since we are not actually using that package). But for now, I guess we can just keep it as `ffmpeg_next`